allennlp_models/generation/dataset_readers/copynet_seq2seq.py

```diff
@@ -194,6 +202,9 @@ def text_to_instance(
 
         fields_dict["metadata"] = MetadataField(meta_fields)
 
+        if weight is not None:
+            fields_dict["weight"] = ArrayField(np.array([float(weight)], dtype=np.single))
```

Comment:
Just make it a `TensorField`, and use Torch methods to create the tensors. `ArrayField` is the old way.